rasterio/enums.py

```diff
@@ -26,8 +26,38 @@ class ColorInterp(IntEnum):
 
 
 class Resampling(IntEnum):
-    """Available warp resampling algorithms.
-
+    """Available warp resampling algorithms are:
+    
+    nearest = 0 : Nearest neighbor resampling (default, fastest algorithm, worst interpolation quality).
+    
+    bilinear = 1 : Bilinear resampling.
```

Comment:
@spestana thanks! My last suggestion is to follow https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring and format these into an "Attributes" section like

```suggestion
    """Available warp resampling algorithms.

    Attributes
    ----------
    nearest :
        Nearest neighbor resampling (default, fastest algorithm, worst interpolation quality).
    bilinear :
        Bilinear resampling.
    ...
```

Reply:
The integer values aren't relevant to the documentation and can be omitted.

Reply:
Thanks! Will do